pages/docs/balance/business-verified-customer/handle-verification-statuses.mdx

```diff
@@ -780,21 +790,21 @@ If the document was found to be fraudulent or doesn’t match the identity of th
-| `ScanIdExpired`           | ID is expired or missing expiration date                               |
-| `ScanIdTypeNotSupported`  | ID may be a military ID, firearm license, or other unsupported ID type |
-| `ScanIdUnrecognized`      | ID is not recognized                                                   |
-| `ScanNameMismatch`        | Scan name does not match name on account                               |
-| `ScanNotReadable`         | Image blurry, too dark, or obscured by glare                           |
-| `ScanNotUploaded`         | Scan not uploaded                                                      |
+A document can fail if, for example, the Customer uploaded the wrong type of document or the `.jpg` or `.png` file supplied was not readable (i.e. blurry, not well lit, not in color, or cuts off a portion of the identifying image). If you receive a `customer_verification_document_failed` webhook, you’ll need to upload another document. To retrieve the failure reason for the document upload, you’ll retrieve the document by its ID. Contained in the response will be a `failureReason` field which corresponds to one or more of the following values. In case of a failure due to multiple reasons, an additional `allFailureReasons` of `reason`s and `description`s is also returned:
+
+| Failure reason            | Description                                                            | Detailed description                                                                                                                                                                                                                                                                                                 |
+| ------------------------- | ---------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `BusinessDocNotSupported` | Business document not supported                                        | The business document provided is not supported for verification. Please request approved [business documentation](#document-types) from the end user for verification.                                                                                                                                              |
+| `BusinessNameMismatch`    | Business name on account does not match document                       | The legal business name listed in the documentation uploaded does not match the Registered Business Name on the account. The account has been moved to [retry](#handling-status-retry) so that the business name listed at the account level can be adjusted to match the business documentation which was uploaded. |
```

Comment:
```suggestion
| `BusinessNameMismatch`    | Business name on account does not match document                       | The legal business name listed in the documentation uploaded does not match the Registered Business Name on the account. The account has been moved to [retry](#handling-retry-status) so that the business name listed at the account level can be adjusted to match the business documentation which was uploaded. |
```